libmamba/src/specs/channel.cpp

```diff
@@ -155,6 +156,11 @@ namespace mamba::specs
         m_platforms = std::move(platforms);
     }
 
+    auto Channel::id() const -> const std::string&
+    {
+        return m_id;
+    }
+
```

Comment:
Should we consider
```cpp
auto operator==(Channel const& lhs, Channel const& rhs) -> bool {
return lhs.id() == rhs.id();
}
```

Reply:
Ok let's revert this change for now, the current Channel's API is not really consistent with that comparison operator (especially because we can change attributes of an already created channel, without having its ID changed.).